icons.tsv

```diff
@@ -47,7 +47,7 @@ offset	name					id
 45		Solus					solus
 46		Void					void
 47		Zorin OS				zorin
-48		Codeberg			    codeberg
+48		Codeberg				codeberg
```

Comment:
Unrelated change :-D

Reply:
It renders different for all people because the width of a tab depends on the user's editor setting.
I would just keep it as it is because otherwise people will start "fighting" about what is the right number of tabs...

<img width="682" alt="Screenshot 2024-09-14 at 09 49 36" src="https://github.com/user-attachments/assets/3cb0ece7-0b95-4260-843c-c60719e1d15d">

Reply:
For example with https://github.com/cameron-wags/rainbow_csv.nvim you can see which icons are aligned based on the color, very handy.

![image](https://github.com/user-attachments/assets/ae2bf356-7d8f-44d3-803c-aebcb7ec0b07)

Reply:
IF we touch this at all, I believe we should correct that for all tabs and not only for one icon?

Obviously in the beginning (i.e. Lukas) edited with a tabwidth of 4. Notice in the images below also how the header line only matches with 4.
Newer icons are a mix, a lot added with tabwith of 8.

I tend to preserve the original author's decision on tabwidth, but we can of course also change it.
But if we touch one occasion, unify all?
And put that change into one dedicated commit.

![image](https://github.com/user-attachments/assets/196f60d2-f340-435e-94f2-4ee8798ea8b5)

![image](https://github.com/user-attachments/assets/5bbe1dfe-acac-4962-a77f-00a1e439bb15)

----

And it was also ME who later added with the wrong tabstop width :unamused: 

![image](https://github.com/user-attachments/assets/7250be1b-3fc3-45b5-a4b2-71dc233c4c4f)

Reply:
Just one related idea, often when I deal with such files I allow comment lines in the good old unix style in the files.
That could also be used to announce the tabwidth ;)

This is just an idea. At least with some such files at Nerdfonts we have that, and with (I believe) all that I create at my 'real work' (example following).

![image](https://github.com/user-attachments/assets/eedb20b5-97c8-4d28-8993-f9eaa686c7eb)

Reply:
I see, could be viable to add a small comment. Another alternative would be to use a `.editorconfig` file:
```.editorconfig
# 4 space indentation
[icons.tsv]
indent_style = space
indent_size = 4
```

![image](https://github.com/user-attachments/assets/efa9a794-cddd-48dc-9915-87c54f6bf45f)